lisa/tools/xfstests.py

```diff
@@ -159,13 +159,37 @@ def get_xfstests_path(self) -> PurePath:
         return tool_path.joinpath("xfstests-dev")
 
     def set_local_config(
-        self, scratch_dev: str, scratch_mnt: str, test_dev: str, test_folder: str
+        self,
+        scratch_dev: str,
+        scratch_mnt: str,
+        test_dev: str,
+        test_folder: str,
+        test_type: str,
+        mount_opts: str = "",
     ) -> None:
         xfstests_path = self.get_xfstests_path()
         config_path = xfstests_path.joinpath("local.config")
         if self.node.shell.exists(config_path):
             self.node.shell.remove(config_path)
         echo = self.node.tools[Echo]
+        if mount_opts:
+            content = "\n".join(
+                [
+                    "[cifs]",
+                    "FSTYP=cifs",
+                    f"TEST_FS_MOUNT_OPTS=''{mount_opts}''",
+                    f"MOUNT_OPTIONS=''{mount_opts}''",
+                ]
+            )
+            echo.write_to_file(content, config_path, append=True)
```

Comment:
This line is shared, can be moved out from the if clause.

Reply:
updated